src/prpy/planning/ompl.py

```diff
@@ -192,38 +221,34 @@ def _SetPlannerRange(self, robot, ompl_args=None):
             # RRT-Connect's range to the same value used for collision
             # detection inside OpenRAVE.
             longest_extent = numpy.max(dof_ranges)
-            ompl_range = conservative_ratio * longest_extent - epsilon
+            ompl_range  = conservative_ratio * longest_extent - self.epsilon
+            ompl_range *= self.multiplier
```

Comment:
The way I had originally envisioned it was `range = (space_resolution * multipler) - epsilon`, but this should work too.